websites/K/Kinopub/presence.ts

```diff
@@ -0,0 +1,254 @@
+const presence = new Presence({
+	clientId: "1054755173198737458",
+});
+
+enum Assets {
+	Play = "https://i.imgur.com/6s4WyWY.png",
+	Pause = "https://i.imgur.com/PrYtpQb.png",
+	Read = "https://i.imgur.com/wPUmqu5.png",
+	View = "https://i.imgur.com/hxvvGUi.png",
```

Comment:
Change it to 512x512px
```suggestion
	View = "https://i.imgur.com/KN6uwcg.png",
```